build.gradle

```diff
@@ -1,23 +1,28 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
+    ext.buildConfig = [
+        "minSdk"    : 21,
+        "compileSdk": 30,
+        "targetSdk" : 30
+    ]
+
+    apply from: "dependencies.gradle"
+
     repositories {
+        google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
-
-        // Required plugins added to classpath to facilitate pushing to Jcenter/Bintray
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1'
-        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
+        classpath packages.bintray
+        classpath packages.gradle
+        classpath packages.kotlin.plugin
+        classpath packages.maven
     }
 }
 
 allprojects {
     repositories {
+        google()
         jcenter()
+        mavenCentral()
     }
 }
```

Comment:
Do you actually need allprojects or subprojects?
subprojects + root = allprojects.

Also, put mavenCentral() above jCenter() because you know why. 
Also, restrict downloading from jCenter only thte packages/modules that are not yet pushed to Maven Central.

Reply:
Good point, it should be subprojects 👍 I also removed `jcenter()` as it's not required here anymore.